modules/ROOT/pages/munit-maven-plugin.adoc

```diff
@@ -5,16 +5,17 @@ endif::[]
 :version-info: 2.0 and newer
 :keywords: munit, testing, unit testing
 :page-aliases: munit-maven-support.adoc, to-set-up-munit-maven-plugin.adoc, faq-working-with-parent-pom.adoc, to-inherit-parent-plugin.adoc, to-ignore-parent-plugin.adoc, to-override-parent-plugin.adoc, munit-maven-plugin-configuration.adoc,to-configure-munit-maven-plugin-maven.adoc
+:page-deployment-options: acb-ide, studio-ide
 
 MUnit provides a Maven plugin that enables you to run your MUnit tests as part of your continuous integration environment.
 
-The following configurations to the `pom.xml` or `settings.xml` files of your Maven installation apply only when running Munit through Maven.
+The following configurations to the `pom.xml` or `settings.xml` files of your Maven installation apply only when running MUnit through Maven.
 
 == Set Up MUnit Maven Plugin
 
-This documentation assumes you have the `<munit.version>` property in your pom.xml.
+You must have the `<munit.version>` property in your pom.xml to set up MUnit Maven Plugin.
```

Comment:
Why is `munit-version` surrounded by angle brackets here? I wondered if it was an XML element but in this section, it's a variable for a value between version tags in  `<version>${munit.version}</version>`.

It is an XML element in another section, but it's a bit confusing to see that here.

Reply:
@dukesphere, according to my understanding, `<munit.version>` is a property. This property is declared as a property in the pom.xml. In this case, you should use angle brackets:
```
    <properties>
        <munit.version>2.3.0</munit.version>
    </properties>
```
Once declared, the plugin calls the property. In this case, the property is written between curly braces:
```
                <plugin>
                    <groupId>com.mulesoft.munit.tools</groupId>
                    <artifactId>munit-maven-plugin</artifactId>
                    <version>${munit.version}</version>
                    <executions>
                        <execution>
                           ...
                          </formats>
                      </coverage>
                    </configuration>
                </plugin>
```

Reply:
@egrave, please, can you confirm my above statement?
If not, let me know and I'll fix the typo.
Thanks!

Reply:
you're right @Crispy-Salesforce

Reply:
Great! Thanks!